react-function-components/src/index.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const CustomButton = <button>Click Me!</button>;
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
 
+const element = CustomButton();
```

Comment:
Here we don't want to call our component (in react we pretty much never call our own components. Instead, we use them to create a react element

```jsx
const element = <CustomButton />;
```